### PR TITLE
[onert] Add float and quant8 kernels for L2Norm

### PR DESCRIPTION
**Pull request by dr-venkman**
Add floating point and asymmetric quantized kernel support for l2 norm 

ONE-DCO-1.0-Signed-off-by: venkat.iyer <venkat.iyer@samsung.com>

### CODE REVIEW

File: tests/nnapi/specs/V1_0/l2_normalization_quant8.mod.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+#
+# Copyright (C) 2017 The Android Open Source Project
+#
```
Comment (hasw7569) [inline tests/nnapi/specs/V1_0/l2_normalization_quant8.mod.py:3]: Please add `Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved` too.

Comment (hasw7569) [reply tests/nnapi/specs/V1_0/l2_normalization_quant8.mod.py:3]: Please change test spec file name if it is not original code from android: `l2_normalization_quant8.mod.py` -> `l2_normalization_quant8_nnfw.mod.py`

Comment (dr-venkman) [reply tests/nnapi/specs/V1_0/l2_normalization_quant8.mod.py]: Thank you for the comments. I have made the necessary changes.

Comment (dr-venkman) [reply tests/nnapi/specs/V1_0/l2_normalization_quant8.mod.py]: @hasw7569 PTAL

**Pull request merged**